### ISSUE TITLE
About the "border effect" in result

### ISSUE THREAD

**Issue by xpeng** [DmitryUlyanov/texture_nets#33]
Hi, i'm training a model with style "van gogh's starry night" and got some result as follows. there are some weird border like effects in result image no matter how i tune the weight or tvloss parameters. i also tried different image size or style size and even with cropped style image, the results' quality is slightly different but none of them the border effect is disappear. 
![wechatimg1](https://cloud.githubusercontent.com/assets/465136/17722693/b6c612fe-6466-11e6-8fe3-77ee323a3e47.jpeg)
different content image with the same effect:
![test1024_style02_48000](https://cloud.githubusercontent.com/assets/465136/17722591/f5d1a0ae-6465-11e6-913b-05ddcc58f397.jpg)

i use these parameters: -model johnson -image_size 512 -style_size 512 -content_weight 10 -style_weight 500 -learning_rate 0.001 -normalize_gradients true -tv_weight 0.000085

i checked the result after training 1000, 5000,10000 and 40000 iters, things didn't change.

has any body meet the same problem? any suggestion is appreciated, thanks!

**Comment by zikzxak** [comment]
Interesting, I had the same results with different parameters and style image: 
![stylized](https://cloud.githubusercontent.com/assets/11982081/17725165/70b6dc0e-6451-11e6-862e-64ba988e6493.jpg)

**Comment by zikzxak** [comment]
Checked with a different model and no borders. Looks like there's some glitch in the Johnson model.

**Comment by DmitryUlyanov** [comment]
This basically happens because of padding. The effect is not that obvious when `num_iteration` is small. What you can do -- is to cut off the borders or extend your test image first and then cut the borders so the original size is preserved.

**Comment by xpeng** [comment]
@zikzxak, yes may be it's the problem of Johnson model
@DmitryUlyanov , thanks. yes this is the trick i use now, it works.

**Comment by zikzxak** [comment]
@DmitryUlyanov Thank you for your quick response, this works :)
